CHANGELOG.md

```diff
@@ -1,6 +1,6 @@
 ### 3.1.2 (Next)
```

Comment:
Lets bump the version to 3.2.0 since we have a new "feature" as part of this PR.